examples/usdTieredSTO.ts

```diff
@@ -1,11 +1,9 @@
 import { USDTieredSTOEvents_3_0_0, BigNumber } from '@polymathnetwork/abi-wrappers';
-import { PolymathAPI } from '../src/PolymathAPI';
-import { FundRaiseType, ModuleName } from '../src';
+import { FundRaiseType, ModuleName, PolymathAPI } from '../src';
```

Comment:
These should be imported from their respective files (`FundRaiseType` and `ModuleName` from types.ts, `PolymathAPI` from PolymathAPI.ts)